src/components/List/index.test.js

```diff
@@ -66,6 +102,37 @@ describe("Suggest", () => {
     expect(onSelect).toHaveBeenCalled()
   })
 
+  it("should render Powered By Google logo for custom items if displayPoweredByGoogle is true given there are no items and no text result", () => {
+    ListFixture = (
+      <List
+        displayPoweredByGoogle={true}
+        items={[]}
+        textNoResults={chance.string()}
```

Comment:
Do not use an external dev for just one random string. Use `Custom no results test` for example